scripts/releaseChangelog.mjs

```diff
@@ -227,6 +227,8 @@ We'd like to offer a big thanks to the ${
 TODO INSERT HIGHLIGHTS
 ${changeLogMessages.length > 0 ? '\n\n' : ''}${changeLogMessages.join('\n')}
 
+<!--/ DO_NOT_REMOVE /-->
```

Comment:
That's the only part I'm not a fan of mostly because it doesn't bring value to whoever is reading the changelog. Maybe we can solve that by having a more meaningful comment if possible.

Reply:
One idea:

```suggestion
<!--/ HIGHLIGHT_SEPARATOR /-->
```